default/scripting/species/SP_SLY.focs.txt

```diff
@@ -18,7 +18,11 @@ Species
 
     defaultfocus = "FOCUS_INDUSTRY"
     likes = ["FOCUS_INDUSTRY" "SPARK_FOSSILS_SPECIAL" "MIMETIC_ALLOY_SPECIAL" "CRYSTALS_SPECIAL" "KRAKEN_NEST_SPECIAL" "PANOPTICON_SPECIAL" "PHILOSOPHER_SPECIAL" "PLC_ARTISAN_WORKSHOPS" "PLC_CONFEDERATION" "PLC_CONTINUOUS_SCANNING" "PLC_EXPLORATION" "PLC_NATIVE_APPROPRIATION" "PLC_DIVERSITY" ]
-    dislikes = ["BLD_GAS_GIANT_GEN" "BLD_BIOTERROR_PROJECTOR" "BLD_SCANNING_FACILITY" "BLD_MEGALITH" "BLD_SPACE_ELEVATOR" "BLD_HYPER_DAM" "BLD_PLANET_BEACON" "BLD_SPATIAL_DISTORT_GEN" "ELERIUM_SPECIAL" "RESONANT_MOON_SPECIAL" "TEMPORAL_ANOMALY_SPECIAL" "SUPERCONDUCTOR_SPECIAL" "PLC_RACIAL_PURITY" "PLC_TERROR_SUPPRESSION" "PLC_CONFORMANCE" "PLC_CHECKPOINTS" ]
+    dislikes = [
+        "SP_THENIAN"          // because they compete for habitat
```

Comment:
acknowledged. as you probably see, i answered the comments from bottom to top .. i'll uncomment all the species likes and dislikes out again. (although that makes me weep a little because it really looked nice in the pedia how everyone dislikes the nightsiders)